compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java

```diff
@@ -90,7 +90,8 @@
     ExpectedClassOrObjectDefID,
     AnonymousFunctionMissingParamTypeID,
     SuperCallsNotAllowedInlineID,
-    ModifiersNotAllowedID;
+    ModifiersNotAllowedID
```

Comment:
You should leave the trailing `,` at the end so the next diff wont change this line
```java
  ...
  AnonymousFunctionMissingParamTypeID,
  SuperCallsNotAllowedInlineID,
  ModifiersNotAllowedID,
  ;
```